gammapy/datasets/map.py

```diff
@@ -2321,37 +2329,293 @@ def peek(self, figsize=(12, 8)):
 
         """
 
+        def make_rgb_transparent(rgba, bg_rgb=(1, 1, 1)):
```

Comment:
Should we move this function and the following 2 in visualization?

Reply:
I agree `adjust_color_limits`, `create_custom_colormap`, and `make_rgb_transparent`
should go in `gammapy.visualization` preferably in a separate PR that go first so its easier to review and we can make them a bit more general.

Reply:
actually adjust_color_limits is a bit misleading function name, as its sole purpose is to determine some reasonable vmin,vmax to be used in event maps. Not sure how useful it would be outside the scope of this peek() since it basically uses both the npred and counts maps. Maybe I could even move the content of the function to peek() directly.